### PR TITLE
Updated changelog for 2.2

### LINKED ISSUE #1416: Update changelog for FDC3 2.2
While most PRs included in FDC3 2.2 contained updates to the FDC3 changelog, some slipped through without one. Update the changelog to include entries for all PRs that affected the standard or NPM lib. 

Issues or PRs in need of changelog entries: https://github.com/finos/FDC3/issues?q=label%3A%22needs+changelog%22+

### PR DESCRIPTION
**Pull request by kemerava**
### Related Issue
resolves #1416 

## Contributor License Agreement

- [x] I acknowledge that a contributor license agreement is required and that I have one in place or will seek to put one in place ASAP.

## Review Checklist

- [ ] **Issue**: If a change was made to the FDC3 Standard, was an issue linked above?
- [x] **CHANGELOG**: Is a *CHANGELOG.md* entry included?
- [ ] **API changes**: Does this PR include changes to any of the FDC3 APIs (`DesktopAgent`, `Channel`, `PrivateChannel`, `Listener`, `Bridging`)?
  - [ ] **Docs & Sources**: If yes, were both documentation (/docs) and sources updated?<br/>
        *JSDoc comments on interfaces and types should be matched to the main documentation in /docs*
  - [ ] **Conformance tests**: If yes, are conformance test definitions (/toolbox/fdc3-conformance) still correct and complete?<br/>
        *Conformance test definitions should cover all **required** aspects of an FDC3 Desktop Agent implementation, which are usually marked with a **MUST** keyword, and  optional features (**SHOULD** or **MAY**) where the format of those features is defined*
  - [ ] **Schemas**: If yes, were changes applied to the Bridging and FDC3 for Web protocol schemas?<br/>
        *The Web Connection protocol and Desktop Agent Communication Protocol schemas must be able to support all necessary aspects of the Desktop Agent API, while Bridging must support those aspects necessary for Desktop Agents to communicate with each other*
    - [ ] If yes, was code generation (`npm run build`) run and the results checked in?<br/>
        *Generated code will be found at `/src/api/BrowserTypes.ts` and/or `/src/bridging/BridgingTypes.ts`*
- [ ] **Context types**: Were new Context type schemas created or modified in this PR?
  - [ ] Were the [field type conventions](https://fdc3.finos.org/docs/context/spec#field-type-conventions) adhered to?
  - [ ] Was the `BaseContext` schema applied via `allOf` (as it is in existing types)?
  - [ ] Was a `title` and `description` provided for all properties defined in the schema?
  - [ ] Was at least one example provided?
  - [ ] Was code generation (`npm run build`) run and the results checked in?<br/>
        *Generated code will be found at `/src/context/ContextTypes.ts`*
- [ ] **Intents**: Were new Intents created in this PR?
  - [ ] Were the [intent name prefixes](https://fdc3.finos.org/docs/intents/spec#intent-name-prefixes) and other [naming conventions & characteristics](https://fdc3.finos.org/docs/intents/spec#naming-conventions) adhered to?
  - [ ] Was the new intent added to the list in the [Intents Overview](https://fdc3.finos.org/docs/intents/spec#standard-intents)?

### CODE REVIEW

**Comment by kemerava** [comment]
@kriswest do we want to add CHANGELOG for #1345 (it was merged into the web, so maybe it would be updated there?) and #1079 (this one I am not sure if we want the CHANGELOG for)?

**Comment by kemerava** [comment]
> #1345 is fine without a change log as it's just part of implementing FDC3 for the Web.
> 
> #1079 does need a change log entry, and I'm amazed I didn't give it one! The first few bullets from the description contain the relevant info.

Done, thanks @kriswest

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,19 +32,24 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
 * FDC3 apps are now encouraged to instantiate their FDC3 interface (DesktopAgent) using the `getAgent()` function provided by the `@finos/fdc3` module. This will allow apps to interoperate in either traditional Preload DAs (i.e. Electron) as well as the new Browser-Resident DAs. ([#1191](https://github.com/finos/FDC3/pull/1191))
+* `Context` and `Intent` are now union type instead of enum, including in functions using intent reference changed from string to `Intent`. ([#1139](https://github.com/finos/FDC3/pull/1139))
 
 ### Deprecated
 
 * Made `IntentMetadata.displayName` optional as it is deprecated. ([#1280](https://github.com/finos/FDC3/pull/1280))
 * Deprecated `PrivateChannel`'s synchronous `onAddContextListener`, `onUnsubscribe` and `onDisconnect` functions in favour of an `async` `addEventListener` function consistent with the one added to `DesktopAgent` in #1207. ([#1305](https://github.com/finos/FDC3/pull/1305))
+* `ContextTypes` and `Intents` enums. ([#1139](https://github.com/finos/FDC3/pull/1139))
```
Comment (kriswest) [inline CHANGELOG.md:41]:
```suggestion
* The `ContextTypes` and `Intents` enums were deprecated in favour of the new ContextType and Intent unions. ([#1139](https://github.com/finos/FDC3/pull/1139))
```

**Review by kriswest — DISMISSED** [review]
LGTM, and thanks for doing this, suggested some clarificaiton on #1139

**Comment by kemerava** [comment]
Thanks for additional suggestions, @kriswest! Added

**Review by kriswest — APPROVED** [review]
LGTM

**Pull request merged**